examples/okta_saml_app/README.md

```diff
@@ -0,0 +1,7 @@
+# okta_saml_app
+
+This resource represents an Okta SAML Application in various configuration states. For more information see the [API docs](https://developer.okta.com/docs/api/resources/apps#add-custom-saml-application)
+
+## Preconfigured Applications
+
+It should be noted that Okta does not provide an API endpoint to configure "preconfigured" SAML apps such as AWS so they can only be created and slightly configured. You cannot enable API Integrations. This is the only known limitation as of now.
```

Comment:
Not sure what this means - so you can create a resource for AWS SAML app but only configure some subset of functionality?

Reply:
Yea I chatted with our Okta liaison and it is not possible to enable or configure API Integrations for certain types of preconfigured SAML apps such as AWS. It is a feature lacking in their API.